Source/WebCore/html/DOMTokenList.h

```diff
@@ -90,4 +90,12 @@ inline const AtomString& DOMTokenList::item(unsigned index) const
     return index < tokens.size() ? tokens[index] : nullAtom();
 }
 
+inline void DOMTokenList::associatedAttributeValueChanged(const AtomString&)
```

Comment:
Maybe we should drop the parameter since it's not used?

Reply:
Sure, we can do that.